package.json

```diff
@@ -161,6 +161,7 @@
     "eslint": "7.23.0",
     "eslint-config-airbnb-typescript": "12.0.0",
     "eslint-config-prettier": "6.11.0",
+    "eslint-config-svg-needs-viewBox": "link:./.eslint/rules",
```

Comment:
Since you made a plugin and not a config, I do not think this is needed?

Reply:
I took it out and installed again and all looks well. Good catch, thank you!